src/bindings/fluid_filerenderer.c

```diff
@@ -289,7 +289,7 @@ new_fluid_file_renderer(fluid_synth_t *synth)
         goto error_recovery;
     }
     
-#ifdef _WIN32
+#if defined( _WIN32 ) && defined( _UNICODE )
```

Comment:
@derselbst : I think that the #if is wrongly placed here. If should be placed just before the `else`.